cqrs-postgres/src/db_wrapper.rs

```diff
@@ -0,0 +1,177 @@
+use crate::util::Sequence;
+use cqrs_core::{CqrsError, EventNumber, RawEvent, Since};
+use num_traits::ToPrimitive;
+use postgres::{rows::Row, types::ToSql};
+use r2d2::{Pool, PooledConnection};
+use r2d2_postgres::PostgresConnectionManager;
+use std::{error, fmt, sync::Arc};
+
+#[derive(Debug, Clone)]
+pub enum DbError {
```

Comment:
We should rename this, as this is no longer just the `DbError`; it also includes errors for the reaction, so change this to `ReactorError<R> where R: Reaction`.

Reply:
Done